### PR TITLE
fixed bug can not pop job from last queue on PHP7.1

### PR DESCRIPTION
**Pull request by davidtall**
see #61

### CODE REVIEW

**Comment by francislavoie** [comment]
Wow, great find. I feel like I might've run into this bug too but never realized what was wrong. @xelan what do you think?

Thanks @davidtall

File: src/Resque/Queue.php [hunk 1]
```diff
@@ -128,8 +128,8 @@ public function pop(array $queues, $timeout = 10, $blocking = true)
     {
         $queue = $payload = null;
 
-        foreach ($queues as &$queue) {
-            $queue = self::redisKey($queue);
+        foreach ($queues as $key=>$queue) {
```
Comment (francislavoie) [inline src/Resque/Queue.php]: Please change it to `$key => $queue` (with spaces) for code style

**Comment by xelan** [comment]
Thank you for your PR, @davidtall and for the review, @francislavoie!
I'll test your changes on a fresh PHP 7.1 system and merge it into 2.1.2 afterwards. Probably this will be done on Tuesday or Wednesday.

**Comment by davidtall** [comment]
see #64

**Comment by francislavoie** [comment]
I'd say it's probably better to put this other bug fix in a different PR, because they're two completely different bugs.

**Comment by davidtall** [comment]
how to cancel the submit?

**Comment by francislavoie** [comment]
You can commit again reverting the previous one. Most git GUI clients have a "Revert commit" feature if you use one, otherwise you can use [git revert](https://git-scm.com/docs/git-revert)

**Comment by francislavoie** [comment]
You reverted the wrong commit 😛

**Comment by davidtall** [comment]
It is OK :😄

**Comment by davidtall** [comment]
So,should I commit the change to another branch and make a PR again?

**Comment by francislavoie** [comment]
Yeah

**Comment by xelan** [comment]
@davidtall Could you please squash your commits here to a single one? Otherwise, it would be really messy in the commit history after merge.

Thank you!

**Comment by davidtall** [comment]
Should I squash my commits to a single one  and make a new PR again?

**Comment by xelan** [comment]
Just squash them to a single commit and force-push to your davidtall/php-resque master branch. A PR is always updated automatically when you push to its base branch :smile:

**Comment by davidtall** [comment]
you said squash #62  and #66  to a single commit and force-push?

**Comment by xelan** [comment]
No, #62  should stay separately. However, here you have six commits now for just one tiny change. Please squash them to one.

For #66, your base branch is the davidtall/php-resque `develop` branch. This is fine, but there you should adapt the code style to PSR-2 (see my review notes) and then also squash those changes to one commit in that branch.

Thank you very much!

**Comment by davidtall** [comment]
Oh No~~Let me try again...

**Comment by davidtall** [comment]
Is it OK ? @xelan

**Comment by xelan** [comment]
Looks good, thank you @davidtall! You could even combine the two commits to a single one, but it's ok. I'll try your PR on a PHP7.1 system later today.

**Comment by davidtall** [comment]
Thanks ,It's my first time to combine commits use `git rebase -i` 😄
